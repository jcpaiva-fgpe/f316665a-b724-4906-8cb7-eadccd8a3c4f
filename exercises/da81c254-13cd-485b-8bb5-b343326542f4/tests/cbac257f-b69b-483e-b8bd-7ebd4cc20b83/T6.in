16 7
23 7
