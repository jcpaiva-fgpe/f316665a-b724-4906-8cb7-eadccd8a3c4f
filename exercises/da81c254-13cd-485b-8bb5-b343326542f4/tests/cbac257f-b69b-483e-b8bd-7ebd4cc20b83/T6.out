Passaram apenas 420 minutos!
Queres dizer, 7 horas?!
